test/unit/containerDeploymentWebviewController.test.ts

```diff
@@ -311,6 +311,9 @@ suite("ContainerDeploymentWebviewController", () => {
 
     test("completeDockerStep reducer updates step status and handles success/failure", async () => {
         const addContainerConnectionStub = sinon.stub(controller as any, "addContainerConnection");

```

Comment:
For another PR, but we generally want to use `sinon.sandbox` instead; set it up in the `setup()` method, and restore it in the `teardown()`.  Every stub that's created in the sandbox gets restored when `sandbox.restore()` is called, so it saves the effort of having to remember the different references.

See `test\unit\addFirewallRuleWebviewController.test.ts` for an example